plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHivePartitionedBucketFunction.java

```diff
@@ -91,18 +92,18 @@ public void testMultiplePartitions(BucketingVersion hiveBucketingVersion)
         BucketFunction hiveBucketFunction = bucketFunction(hiveBucketingVersion, numBuckets, ImmutableList.of(HIVE_LONG));
 
         int numPartitions = 8;
-        ImmutableList.Builder<Long> partitionValuesBuilder = ImmutableList.builder();
-        for (int i = 0; i < numPartitions; i++) {
-            partitionValuesBuilder.addAll(Collections.nCopies(numValues / numPartitions, i * 348349L));
+        List<Long> partitionValues = new ArrayList<>();
```

Comment:
unrelated. right?

Reply:
It's used to generate `null` partition. This is the way I discovered that 0 was bad hash code (not all buckets generated).